web/tools.html

```diff
@@ -156,21 +167,21 @@ <h4>Tools</h4>
         </div>
 
         <footer>
-            <!--Buttons from RERUM Home Page. Not properly reflexive currently, and missing the correct font.-->
             <div class="footerButtons">
-                <a href="https://rerum.io/#/welcome" class="button">
-                    Home
-                </a>
                 <a href="https://rerum.io/#/annotation" class="button">
+                    <img src="/web/images/rerum_logo.png" alt="Annotation Icon">
```

Comment:
These icons appear to have a special class in a style sheet on the main RERUM page. Is there a way we can find the original source of the images from that, or load the same styles ourselves?

Reply:
You can find the images in this repository [RERUM](https://github.com/CenterForDigitalHumanities/rerum)